.github/workflows/release.yml

```
@@ -0,0 +1,52 @@
+name: Create release candidate
+
+on: [workflow_dispatch]
+
+jobs:
+  create_release:
+    name: Create release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/cache@v3
+        with:
+          path: ./node_modules
+          key: npm-cache-${{ runner.os }}-${{ hashFiles('./package-lock.json') }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Run release script and open PR
+        run: |
+          git config user.name "API Team CI User"
+          git config user.email "aciattestteamci@deque.com"
+
+          Branch="release-$(date +%Y-%m-%d)"
+          echo "Branch: $Branch"
+          git checkout -b "$Branch"
+
+          npm ci
+          npm run release
+
+          git push origin "$Branch" --force
+
+          Version=$(jq -r .version ./package.json)
+          echo "Version: $Version"
+
+          # Stolen from https://github.com/dequelabs/axe-extension/blob/0f22963c4d0a73bf853aa982ade2393e422f002c/scripts/release.sh#L51-L54
```

This is a private url, shouldn't list it in public repo.

```suggestion
          # Get the additions to the changelog as the commit body and generate the PR url
```

---

Fair enough 🤷